s1m0nWasTaken/CS4337_BigDataProject
# Mo/user service test

## Summary
I added some more test cases for user services. I also added Jacoco dependencies into gradle file. They all seem to pass and I have gotten 100 percent test coverage so all of them are getting run. I was having trouble with testAddUserWithNullUserType() but it started passing as well, I am not sure why. 

## How to verify
I used ./gradlew :UserService:test to run the userService tests exclusively and ran .\gradlew jacocoTestReport to generate code coverage. You should be able to access it from the path CS4337_BigDataProject\UserService\build\reports\tests\test\index.html. Let me know if you want me to change or add anything.

## Tests
testBanUserWithoutId
testBanUserAlreadyBanned
testDeleteUserNonNumericID
testDeleteNonExistentUser
Are some of the test cases

## Related
